src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,118 @@
+.contact {
```

Comment:
![Screen Shot 2022-12-21 at 8 14 18 PM](https://user-images.githubusercontent.com/118940236/208977203-5d1ebda4-a262-401c-9703-a5247530e1b8.png)
The form should stretch across the width and the contact text should be aligned to the left side